src/core/inlines.js

```diff
@@ -70,66 +146,18 @@ export function run(conf) {
       if (subtxt.length) matched = subtxt.shift();
       df.appendChild(document.createTextNode(t));
       if (matched) {
```

Comment:
Can probably even return early here... saves one level of code nesting.

Reply:
missed out on this review. I'll do it in the next set of changes :)